resolver/pkg/resolver.go

```diff
@@ -79,8 +79,19 @@ type imageTagTransformer struct {
  then loops though all images inside containers session, finds matched ones and update the tag name
 */
 func (pt *imageTagTransformer) findAndReplaceTag(obj map[string]interface{}) error {
-	paths := []string{"containers", "initContainers"}
 	found := false
+
+	// Update contain.image
+	_, found = obj["container"]
```

Comment:
Please combine it with `for _, path := range paths` loop. I don't see the reason why this code is separate.

Reply:
because one path is a list and other path is a single map.